docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -130,4 +130,41 @@ When you attach an indicator to a case, the indicator is added as a new comment
 To remove an indicator attached to a case, click *More actions* (*...​*) -> *Delete attachment* in the case comment. 
 
 [role="screenshot"]
-image::images/remove-indicator.png[Removing an indicator from a case]
\ No newline at end of file
+image::images/remove-indicator.png[Removing an indicator from a case]
+
+[discrete]
+[[add-indicator-to-blocklists]]
+== Add indicators to blocklists
+
+Add indicators to blocklists to prevent selected applications from running on your hosts. You can add indicators of the `file` indicator type and that have hash values for the MD5, SHA-1, or SHA-256 fields.
+
+.Requirements
+[sidebar]
+--
+* In addition to configuring specific entries on the **Blocklist** page, you must also ensure that the blocklist is enabled on the {elastic-defend} integration policy in the <<malware-protection, Malware protection settings>>. This setting is enabled by default.
+
+* You must have the `superuser` {ref}/built-in-roles.html[built-in user role] to access this feature.
+--
+
+To add an indicator to a blocklist:
+
+. From the Indicators table, click the *More actions menu* (*...​*) menu, then select *Add blocklist entry*. Alternatively, open an indicator's details, then select *Take action > Add blocklist entry*. The *Add blocklist* flyout opens.
+
+. Fill in these fields in the **Details** section:
+.. `Name`: Enter a name to identify the indictor in the blocklist.
+.. `Description`: Enter a description to provide more information on the blocklist entry (optional).
+
+. In the **Conditions** section, verify indicator details are correct.
```

Comment:
"That" is sometimes an optional connector word; in this case I think helps avoid a more clipped tone.
```suggestion
. In the **Conditions** section, verify that indicator details are correct.
```

Reply:
I have no strong feelings one way or another but good point :)